### PR TITLE
SPyify more tutorials

### LINKED ISSUE #2899: Python tutorials don't offer much guidance outside of hints
A lot of the python tutorials are difficult to follow unless you already know the APIs. For example, this is the second step in the flashing heart tutorial:

Screen Shot 2020-04-24 at 10 55 53 AM

It doesn't say how to create a template string, or that show leds is in the basic category, or that you "draw" by using the characters "#" and ".". The only real way to move forward is to look at the hint.

### PR DESCRIPTION
**Pull request by ganicke**
Make `spy` versions of:

- [x] Coin Flipper
- [x] Tug of LED
- [x] Flashing Heart
- [x] Name Tag
- [x] Smiley Buttons
- [x] Dice
- [x] Love Meter
- [x] Micro Chat
- [x] 7 Seconds
- [x] Heads Guess
- [x] Hot Potato
- [x] Snap the Dot
- [x] Level
- [x] Stopwatch
- [ ] Compass

RE: #2880
Closes #2899

### CODE REVIEW

**Comment by ganicke** [comment]
Some considerations for step descriptions:

- Use explicit hints for all `spy` versions?
- Keep the 'suggestive' style introduced by @pelikhan or call out APIs more directly by name/use?

**Comment by pelikhan** [comment]
* yes always visible.
* Subjective style work well in minecraft. Trying to describe text is worse than just showing text.

File: docs/projects/spy/7-seconds.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+Compute the elapsed time as ``||input:running time||`` ``||math:minus||`` ``||variables:start||`` and
+store it into a new variable called ``||variables:elapsed||``.
+
+```spy
+let start = 0
+let elapsed = 0
+input.onButtonPressed(Button.B, function () {
+    // @highlight
+    elapsed = input.runningTime() - start
```
Comment (pelikhan) [inline docs/projects/spy/7-seconds.md:72]: we can use local variables when applicable in spy

**Comment by pelikhan** [comment]
@ganicke  i've added the compass :)

**Pull request merged**